junit5/src/main/java/org/jboss/weld/junit5/ExtensionContextUtils.java

```diff
@@ -16,15 +16,14 @@
  */
 package org.jboss.weld.junit5;
 
+import java.util.List;
+
 import org.jboss.weld.environment.se.WeldContainer;
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
 
-import java.util.List;
-
-
 /**
- * Utility methods for fetching/storing items stored in the JUnit extension context.
+ * Utility methods for fetching/retrieving items stored in the JUnit extension context store.
```

Comment:
I would add a note that this util class is not supposed to be part of the API, although it's public...